rosa_planning/rosa-hcp-prereqs.adoc

```diff
@@ -0,0 +1,87 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/attributes-openshift-dedicated.adoc[]
```

Comment:
I cannot see this content in any of the provided links. I'll continue to review the module, but the module should be included in either the ROSA or HCP docs?